premium/usage.go

```diff
@@ -240,6 +260,21 @@ func NewUsageClient(meta plugin.Meta, ops ...UsageClientOptions) (UsageClient, e
 	return u, nil
 }
 
+func isAWSMarketplace() bool {
+	return os.Getenv("CQ_AWS_MARKETPLACE_CONTAINER") == "true" || os.Getenv("CQ_AWS_MARKETPLACE_AMI") == "true"
```

Comment:
Who's responsible for setting these? AWS?

Reply:
We are, we set it in the container we upload to the marketplace